src/libratbag-enums.h

```diff
@@ -327,3 +327,22 @@ enum ratbag_macro_event_type {
 	RATBAG_MACRO_EVENT_KEY_RELEASED,
 	RATBAG_MACRO_EVENT_WAIT,
 };
+
+/**
+ * @ingroup enums
+ *
+ * Device Types
+ * Describes the types specified in the .device files
+ */
+enum ratbag_device_type {
+	// Used when no DeviceType property could be found
+	TYPE_UNSPECIFIED = 0,
+	// Used for mice
+	TYPE_MOUSE,
+	// Used for Keyboards
+	TYPE_KEYBOARD,
```

Comment:
Just for consistency, if you write "Keyboard" with a capital letter in the beginning, do it for "mice" as well.

Reply:
ok